src/components/SidePanes/UsersPane.jsx

```diff
@@ -119,6 +121,7 @@ class UsersPane extends React.Component {
             ))}
           </div>
         }
+        {!showEditors && organizationUsers.error && <div className="error-msg">{organizationUsers.error[0]}</div>}
```

Comment:
We have a ```StatusAlert``` component I think we've used elsewhere for showing errors, might be worth just using that guy instead.

Reply:
Also might be worth checking if the len of organizationUsers.error > 0 as well just for safety